backend/tests/data/test_sql_alchemy.py

```diff
@@ -1,17 +1,13 @@
-from unittest.mock import Mock
 import uuid
 from pathlib import Path
 
 import pytest
-import pytest_asyncio
-from sqlalchemy.ext.asyncio import create_async_engine
-from sqlalchemy import text
-
-from chainlit.data.base import BaseDataLayer, BaseStorageClient
+from chainlit import User
+from chainlit.data.base import BaseStorageClient
 from chainlit.data.sql_alchemy import SQLAlchemyDataLayer
 from chainlit.element import Text
-from chainlit import User
-from chainlit.user import PersistedUser
+from sqlalchemy import text
+from sqlalchemy.ext.asyncio import create_async_engine
```

Comment:
See earlier comments on preferred import order. If we're not following that, I'd like to hear your suggestion.
It's a good time to start getting consistent on this (more important than 'my own way'). :)

Reply:
That is the result of running the Trunk formatter, which uses ruff, isort and black to organize the imports. I'm trying to move everything to be consistent. It is not my own way.